EI patterns/document_message/main.bal

```diff
@@ -0,0 +1,18 @@
+import ballerina/http;
+import ballerina/mime;
+
+final http:Client zohoClient = check new ("http://content.zohoapis.com.balmock.io", retryConfig = {count: 3, interval: 1, statusCodes: [404, 408, 500]}
+);
+
+listener http:Listener httpListener = new (port = 8080);
+
+service /crm on httpListener {
+    resource function post bulkUploadLeads(CsvRequest csvRequest) returns ZohoResponse|error {
+        http:Request request = new http:Request();
+        () var1 = request.addHeader("X-CRM_ORG", csvRequest.org);
```

Comment:
```suggestion
        _ = request.addHeader("X-CRM_ORG", csvRequest.org);
```

Is that possible to change like this?

Reply:
Are we allowing this from the BI side? My concern is that if we're working from the UI side, would this be acceptable? Normally, this would be considered best practice in coding, but I believe this isn't something we would do in the UI, correct?